src/js/comments.js

```diff
@@ -27,13 +27,14 @@ class Comments {
 	_renderComments () {
 		/*global Coral*/
 		getJsonWebToken()
-			.then(token => {
+			.then(({ token, userIsSignedIn }) => {
 				const scriptElement = document.createElement('script');
-				scriptElement.src = 'https://ft-next-talk-spike.herokuapp.com/static/embed.js';
+				scriptElement.src = 'https://92fb06bc.ngrok.io/static/embed.js';
```

Comment:
This all looks fab. Just one question...how does this bit work?

Reply:
I forgot to revert my local testing URLs. Thanks for spotting this.